app-creator/src/widgets/ui-button/ui-button.ts

```diff
@@ -51,6 +54,22 @@ export class Button extends LitElement {
     `;
   }
 
+  setAttribute(key: string, value: string) {
+    switch (key) {
+      case 'label':
+        this.label = value;
+        break;
+      case 'raised':
+        this.raised = value === 'true';
+        break;
+      case 'disabled':
+        this.disabled = value === 'true';
+        break;
```

Comment:
Would it be possible to have an actual boolean `value` so we don't have to compare the strings?

Reply:
+1, supporting multiple types is a good idea for extensibility.

Reply:
We will address this in a future PR. Issue tracked here https://github.com/googleinterns/earthengine-community/projects/1#card-39816574.